e2e/app/element/container.ts

```diff
@@ -0,0 +1,19 @@
+export default class Container {
+
+  private readonly _xpath: string;
+
+  constructor(selector: {xpath?: string, testId?: string}) {
```

Comment:
Is it possible to have factory methods, one taking an xpath and another taking  testId? Or get the xPath given a testId and just continue on?

Having two constructor parameters that are mutually exclusive is a confusing pattern, especially when both of them look like strings.

If you had a helper `testIdToXPath(testId?: string)` you could call that before constructing one of these.